features/bevragen/persoon/verblijfplaats/dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,268 @@
+# language: nl
+
+Functionaliteit: vertalen van gevraagde samengestelde of afgeleide verblijfplaats velden naar leveren van bron velden
+  Wanneer met fields wordt gevraagd om een veld dat wordt samengesteld uit verschillende andere velden in de GbaPersoon of als een veld dat moet worden meegeleverd wordt samengesteld uit verschillende andere velden in de GbaPersoon, dan worden deze benodigde velden geleverd.
+
+  Rule: wanneer een veld van verblijfplaats m.b.v. de alias 'verblijfplaatsBinnenland' wordt gevraagd worden alleen de velden die horen bij een binnenlandse verblijfplaats geleverd die nodig zijn voor het bepalen van het type binnenlands verblijfplaats.
```

Comment:
Ik vraag mij af of de rules niet moeten zijn:
- als verblijfplaats of één of meerdere velden van verblijfplaats wordt gevraagd en de consumer is hiervoor geautoriseerd, dan worden alle verblijfplaats velden geleverd
- als verblijfplaatsBinnenland of één of meerdere velden van verblijfplaatsBinnenland wordt gevraagd en de consumer is hiervoor geautoriseerd, dan worden alle verblijfplaats velden van Adres en/of Locatie geleverd